src/dbnode/storage/database.go

```diff
@@ -1141,6 +1141,12 @@ func (d *db) AggregateTiles(
 	targetNsID ident.ID,
 	opts AggregateTilesOptions,
 ) (int64, error) {
+	opts.MetricsScope = d.scope.SubScope("computed-namespace").
```

Comment:
This should probably be set when `opts` are created rather than mutating incoming options; would make it much easier to find where this is set